paasta_tools/utils.py

```diff
@@ -2097,7 +2097,7 @@ def get_slack_token(self) -> str:
         return self.config_dict.get('slack', {}).get('token', None)
 
     def get_tron_config(self) -> dict:
-        return self.config_dict.get('tron', {})
+        return self.config_dict.get('tron_api_endpoints', {})
```

Comment:
why do we need to change this?

Reply:
Since I changed the schema, I just think this makes more sense.